### ISSUE TITLE
Consistent undefined comic id on the library view

### ISSUE THREAD

**Issue by mcpierce** [mcpierce/comixed#185]
**Describe the bug**
While it doesn't seem to cause a performance problem or error, I keep seeing the following error in the console on Chrome:

```
GET http://localhost:7171/api/comics/undefined/pages/0/content 400 ()
```

**To Reproduce**
1. Go to the home page while using Google Chrome.
1. Open the browser's console view.
1. Go to the **Library** view.

**Expected behavior**
No error messages in the console.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Google Chrome
 - Version 68.0.3440.106 (Official Build)(64- bit)

**Additional context**
I have not tested on Chrome for other operating systems.

**Comment by awe247** [comment]
Hi, I think that this might just be because the **current_comic** value in the _ComicService_ is being initially set with an unpopulated structure...

`current_comic: Subject<Comic> = new BehaviorSubject<Comic>(new Comic());`

Then the undefined **id** of this object makes it's way through to the api requests and causes the error above. It only goes that way because _comic.missing_ is also undefined (and therefore not true).

So a simple fix might be to initialize the current_comic value with a missing value of true until its set to something meaningful?

**Comment by mcpierce** [comment]
Fixed by #243.
